packages/rocketchat-spotify/lib/spotify.js

```diff
@@ -0,0 +1,87 @@
+/*
+ * Spotify a named function that will process Spotify links or syntaxes (ex: spotify:track:1q6IK1l4qpYykOaWaLJkWG)
+ * @param {Object} message - The message object
+ */
+
+let process = undefined;
+class Spotify {
+	static initClass() {
+		process = function(message, source, callback) {
+			if (_.trim(source)) {
+				// Separate text in code blocks and non code blocks
+				const msgParts = source.split(/(```\w*[\n ]?[\s\S]*?```+?)|(`(?:[^`]+)`)/);
+
+				return (() => {
+					const result = [];
```

Comment:
You can remote this return and result.
Coffee always return the last line, but we are not using the return of this method here